sphinxserve/__init__.py

```diff
@@ -48,6 +52,13 @@
                         short: d
                         action: store_true
                         default: __SUPPRESS__
```

Comment:
:) Absolutely, lets meet there. Pycon is one incredible space where so many excellent projects have been born. The talks are amazing and the community incredible friendly and supportive. If you can, try to be on the sprints too. Sharing time with projects, their communities and leaders is truly invaluable.